static/scss/partials/main.scss

```diff
@@ -55,7 +55,7 @@ body {
   display: flex;
   flex-direction: column;
   margin: 0;
-  padding: 83px 0 0 0;
+  padding: 0 0 0 0;
```

Comment:
Just wondering if we need this line if all the values are set to zero?

Reply:
I'll check – if anything, it might be a `padding: 0` only.

Reply:
Fixed!